packages/components/text/README.md

```diff
@@ -100,19 +100,20 @@ import Text from '@commercetools-uikit/text';
-| `isItalic`        | `Boolean`        |    -     | -                                                                             | `false` |                                                                                           |
-| `isStrikethrough` | `Boolean`        |    -     | -                                                                             | `false` |                                                                                           |
-| `tone`            | `String`         |    -     | `['primary', 'secondary', 'information', 'positive', 'negative', 'inverted']` | -       |                                                                                           |
-| `children`        | `PropTypes.node` | ✅ (\*)  | -                                                                             | -       |                                                                                           |
-| `intlMessage`     | `intl message`   | ✅ (\*)  | -                                                                             | -       | An `intl` message object that will be rendered with `FormattedMessage`                    |
-| `title`           | `String`         |    -     | -                                                                             | -       |                                                                                           |
-| `truncate`        | `Bool`           |    -     | -                                                                             | `false` |                                                                                           |
-| `nowrap`          | `Bool`           |    -     | -                                                                             | `false` | The content in the element will not be wrapped to a new line unless explicitly specified. |
+| Props             | Type             | Required | Values                                                                        | Default   |                                                                                           |
+| ----------------- | ---------------- | :------: | ----------------------------------------------------------------------------- | --------- | ----------------------------------------------------------------------------------------- |
+| `as`              | `String`         |    -     | `['p', 'span']`                                                               | -         |                                                                                           |
+| `id`              | `String`         |    -     | -                                                                             | -         | Used as HTML id property                                                                  |
+| `isBold`          | `Boolean`        |    -     | -                                                                             | `false`   |
```

Comment:
We should also add a warning here about `isBold` is deprecated in favour of `fontWeight`.